contracts/bondingcurve/CommonsToken.sol

```diff
@@ -147,6 +148,12 @@ contract CommonsToken is BondingCurveToken, Pausable {
     minExternalContribution = _settings[7];
 
     externalToken = ERC20(_addresses[0]);
+
+    initialRaiseInternal = _calcInternalTokens(initialRaise);
```

Comment:
shouldn't this be 0 when a fresh contract is deployed? and generated once the hatch contribution happens?

Reply:
Yes, agreed. This should be calculated during _endHatchPhase()